cli/src/semgrep/formatter/sarif.py

```diff
@@ -326,6 +346,9 @@ def _rule_to_sarif_tags(rule: Rule) -> Sequence[str]:
                 if isinstance(owasp, list)
                 else [f"OWASP-{owasp}"]
             )
+        if "confidence" in rule.metadata:
```

Comment:
```suggestion
        if rule.metadata.get("confidence"):
```
^ uber-nit but lets guard against false-y values like ""